src/Uno.UI.Runtime.Skia.WebAssembly.Browser/ts/Runtime/ImageLoader.ts

```diff
@@ -0,0 +1,43 @@
+namespace Uno.UI.Runtime.Skia {
+	export class ImageLoader {
+		public static async loadFromArray(array: Uint8Array): Promise<object> {
+			return new Promise<object>((resolve) => {
+				const image = new Blob([array]);
+				const imageUrl = URL.createObjectURL(image);
+				const canvas = document.createElement('canvas');
+				const ctx = canvas.getContext('2d')
+
+				const img = new Image();
+				img.onload = () => {
+					URL.revokeObjectURL(imageUrl);
+
+					canvas.width = img.width;
+					canvas.height = img.height;
+					ctx.drawImage(img, 0, 0);
+
+					const imageData = ctx.getImageData(
+						0, 0,
+						canvas.width,
+						canvas.height
+					);
+
+					canvas.width = 0;
+					canvas.height = 0;
+
+					resolve({
+						error: null,
+						bytes: Array.from<number>(imageData.data),
```

Comment:
Let's use this instead:

https://github.com/unoplatform/uno/blob/dd2e98d0bbf0c3c4a23cb1296f4721595621fb03/src/Uno.UWP/ts/Windows/Storage/AssetManager.ts#L17-L19

Used as follows:

https://github.com/unoplatform/uno/blob/master/src/Uno.UWP/Storage/Helpers/AssetsManager.wasm.cs#L55-L78

Reply:
.NET does it already properly.